fenix/app/src/main/java/org/mozilla/fenix/shopping/store/ReviewQualityCheckStore.kt

```diff
@@ -161,5 +161,31 @@ private fun mapStateForUpdateAction(
                 }
             }
         }
+
+        is ReviewQualityCheckAction.UpdateAnalysisProgress -> {
+            state.mapIfOptedIn {
+                when (it.productReviewState) {
+                    is ProductReviewState.NoAnalysisPresent -> {
+                        it.copy(
+                            productReviewState = it.productReviewState.copy(
+                                progress = action.progress.toFloat() / 100,
```

Comment:
We're doing this twice, so maybe we can extract this as a private function, something like `fun Double.normalizePercentage(): Float` so the usage becomes `action.progress.normalizePercentage()`